node/flatpak-node-generator.py

```diff
@@ -1877,6 +1882,9 @@ async def main() -> None:
     parser.add_argument('--electron-node-headers',
                         action='store_true',
                         help='Download the electron node headers')
+    parser.add_argument('--electron-from-rcfile',
+                        action='store_true',
+                        help='Download electron version per node headers from .yarnrc/.npmrc')
```

Comment:
nit: `electron version corresponding to the node headers...`? I had to do a double-take with the string as written

Reply:
Is it better now?

Reply:
yup!